cmd/graphite-writer-stats/graphite-writer-stats.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"flag"
+	"github.com/melchiormoulin/graphite-writer-stats/input"
+	"github.com/melchiormoulin/graphite-writer-stats/output"
+	"go.uber.org/zap"
+)
+
+var (
+	brokers      = "localhost:9092"
```

Comment:
You can just initialize your variables by flag.<Type> instead of flag.<Type>Var, they will get filled at flag.Parse() call

Reply:
It will avoid duplication